packages/flutter_test/pubspec.yaml

```diff
@@ -7,10 +7,8 @@ dependencies:
   # here we pin it precisely to avoid version skew across our packages.
   test: 0.12.13+1
 
-  # We don't actually depend on 'analyzer', but 'test' does. We pin the
-  # version of analyzer we depend on to avoid version skew across our
-  # packages.
-  analyzer: 0.27.2
+  # Pinned in flutter_tools as well.
+  analyzer: 0.27.4-alpha.1
```

Comment:
You're right.  Updated to say:

```
  # We don't actually depend on 'analyzer', but 'test' and 'flutter_tools' do.
  # We pin the version of analyzer we depend on to avoid version skew across our
  # packages.
```